pkg/v13patch1/chartconfig/mock_test.go

```diff
@@ -0,0 +1,25 @@
+package chartconfig
+
+import (
+	"context"
+
+	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/helmclient"
+	"k8s.io/client-go/kubernetes"
+)
+
+type tenantMock struct {
```

Comment:
Would it make sense to have a fake implementation like that in `github.com/giantswarm/tenantcluster`? I feel like these things get repeated everywhere and could simply be used from a common place.